### DISCUSSION TITLE
Why everything is "weeks ago"? Is everybody stop using Github?

### DISCUSSION THREAD

**Discussion by linonetwo** [community/community#68230] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![图片](https://github.com/community/community/assets/3746270/8fe604ce-6da4-40eb-80fd-35632b3ae4a5)

My filter is:

![图片](https://github.com/community/community/assets/3746270/f5e54a1f-84a7-4a54-b122-5e9a1e19af7f)
![图片](https://github.com/community/community/assets/3746270/647a7a47-0f47-4e99-b194-318464242093)

**Comment by spenserblack** [comment]
Check out the old feed at https://github.com/dashboard-feed. You will probably see more events. The new feed is missing some type of events.
